spec/features/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -8,28 +8,240 @@
     allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
   end
 
-  it 'validates zip code input', allow_browser_log: true do
-    user = user_with_2fa
-
-    sign_in_and_2fa_user(user)
-    begin_in_person_proofing(user)
-    complete_prepare_step(user)
-    complete_location_step(user)
-    expect(page).to have_current_path(idv_in_person_step_path(step: :state_id), wait: 10)
-    fill_out_state_id_form_ok(same_address_as_id: true)
-    # blank out the zip code field
-    fill_in t('in_person_proofing.form.state_id.zipcode'), with: ''
-    # try to enter invalid input into the zip code field
-    fill_in t('in_person_proofing.form.state_id.zipcode'), with: 'invalid input'
-    expect(page).to have_field(t('in_person_proofing.form.state_id.zipcode'), with: '')
-    # enter valid characters, but invalid length
-    fill_in t('in_person_proofing.form.state_id.zipcode'), with: '123'
-    click_idv_continue
-    expect(page).to have_css('.usa-error-message', text: t('idv.errors.pattern_mismatch.zipcode'))
-    # enter a valid zip and make sure we can continue
-    fill_in t('in_person_proofing.form.state_id.zipcode'), with: '123456789'
-    expect(page).to have_field(t('in_person_proofing.form.state_id.zipcode'), with: '12345-6789')
-    click_idv_continue
-    expect(page).to have_current_path(idv_in_person_ssn_url)
+  context 'when visiting state id for the first time' do
+    it 'displays correct heading and button text', allow_browser_log: true do
+      complete_steps_up_to_location_step
+
+      expect(page).to have_content(t('forms.buttons.continue'))
+      expect(page).to have_content(t('in_person_proofing.headings.state_id_milestone_2'), wait: 10)
```

Comment:
```suggestion
    expect(page).to have_content(t('in_person_proofing.headings.state_id_milestone_2').tr(' ', ' '))
```

That first one is a non-breaking space, which is different. I saw this being done somewhere else and tried it.

I still think something is fundamentally broken in the chain here, because you're testing for _exactly_ what is being rendered, but somehow... the space is different.

I wonder if we need a `have_content_that_a_reasonable_person_would_consider_identical` helper that does this for us?